test_tools/sigma_verify_rules.py

```diff
@@ -104,9 +159,12 @@ def move_problematic_rule(filepath, move_to_path, reason=None):
             file_objec.write(f'{filepath}\n{reason}\n\n')
 
         base_path = os.path.basename(filepath)
+        logging.info('Moving the rule: {0:s} to {1:s}'.format(
+        filepath, f'{move_to_path}{base_path}'))
         os.rename(filepath, f'{move_to_path}{base_path}')
-    except OSError:
-        logger.error('OS Error - no rules moved')
+    except OSError as e:
```

Comment:
```suggestion
    except OSError:
        logger.error('OS Error - rule not moved', exc_info=True)
```

then you don'\t need the second logger command and don't need to specify a variable for the exception